src/ai/backend/manager/api/admin.py

```diff
@@ -77,17 +81,20 @@ async def _handle_gql_common(request: web.Request, params: Any) -> ExecutionResu
         registry=root_ctx.registry,
         idle_checker_host=root_ctx.idle_checker_host,
     )
-    result = await app_ctx.gql_schema.execute_async(
-        params["query"],
-        None,  # root
-        variable_values=params["variables"],
-        operation_name=params["operation_name"],
-        context_value=gql_ctx,
-        middleware=[
-            GQLLoggingMiddleware(),
-            GQLMutationUnfrozenRequiredMiddleware(),
-            GQLMutationPrivilegeCheckMiddleware(),
-        ],
+    result = await asyncio.wait_for(
```

Comment:
Now `async with asyncio.timeout(): ...` is the recommended timeout pattern.
https://docs.python.org/3/library/asyncio-task.html#timeouts
Also, `wait_for()` raises `TimeoutError` instead of `asyncio.TimeoutError` as of Python 3.11.